posthog/utils.py

```diff
@@ -899,12 +900,8 @@ def str_to_bool(value: Any) -> bool:
     return str(value).lower() in ("y", "yes", "t", "true", "on", "1")
 
 
-def print_warning(warning_lines: Sequence[str], *, top_emoji="🔻", bottom_emoji="🔺"):
-    highlight_length = min(max(map(len, warning_lines)) // 2, shutil.get_terminal_size().columns)
-    print(
-        "\n".join(("", top_emoji * highlight_length, *warning_lines, bottom_emoji * highlight_length, "",)),
-        file=sys.stderr,
-    )
+def print_warning(warning_lines: List[str]):
```

Comment:
Follow up PR, replace all of those with direct calls to `logger.warning`

Reply:
I might just do that now, I didn't want to change too much in one go but I think it makes sense.